### ISSUE TITLE
Metrics work for SPARK PMF (Grafana)

### ISSUE THREAD

**Issue by MF416** [CheckerNetwork/proj-mgmt#66]

**Comment by coreymjames** [comment]
@MF416 is this owned by you, or should others be involved?

Also want to make sure you see these:

- [SPARK Grafana Dashboard](https://protocollabs.grafana.net/d/c4bd3c80-9360-4ea3-8b75-c5ecea90765b/spark?orgId=1)
- [Station Grafana Dashboard](https://protocollabs.grafana.net/d/F2UmmYnVk/station?orgId=1)

**Comment by MF416** [comment]
Yeah this has been lower prio / I was out of commission last week --> happy for others to provide input as well
